utils/priority_queue.mli

```diff
@@ -0,0 +1,40 @@
+(** Max priority queue  *)
+
+[@@@ocaml.warning "+a-40-41-42"]
+
```

Comment:
(Like on #3875, I think we should leave
a comment/CR about the use of a stdlib
module - `Pqueue`.)

Reply:
I put a CR in `priority_queue.ml`